gnosis/safe-react
# Bugfix - Sentry error

## What it solves

This PR closes #2302. (`TypeError: Cannot read property 'height' of undefined `)

## How this PR fixes it

After some research, I realized this error was thrown due to not to have Trust provider on the list, causing `WALLET_ICONS[provider]` to be defined. So I added it as a new provider.

## How to test it
1. Open connect to a wallet popup on web mobile
2. Select Trust
3. You should be able to connect without any problems and see the icon next to your address.
